### PR TITLE
Fix/header section

### PR DESCRIPTION
**Pull request by iamzilfa**
# Description

Fixing the space in the header section 

Fixes # (issue)

-  Space between the logo and paragraph in all breakpoints 
- Margins  on both sides,
-  Removed arbitrary values

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [x] npm run build
- [x] npm run dev

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent

### CODE REVIEW

File: public/assets/images/arrow_small.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="13.36" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (esther921) [inline public/assets/images/arrow_small.svg:1]: I suggest you create one component in which you return all the arrows that will be needed and we can be exporting them! This is an example to it!

`import React from "react";

export const ArrowPointingRight = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      stroke="%23FFFFFF"
      stroke-width="2"
      stroke-linecap="round"
      stroke-linejoin="round"
    >
      <path d="M5 12h13M12 5l7 7-7 7" />
    </svg>
  );
};

export const ArrowPointing = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      stroke="%23FFFFFF"
      stroke-width="2"
      stroke-linecap="round"
      stroke-linejoin="round"
    >
      <path d="M5 12h13M12 5l7 7-7 7" />
    </svg>
  );
};
`

Comment (esther921) [inline public/assets/images/arrow_small.svg:1]: I suggest you create one component in which you return all the arrows that will be needed and we can be exporting them! This is an example to it!

`import React from "react";

export const ArrowPointingRight = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      stroke="%23FFFFFF"
      stroke-width="2"
      stroke-linecap="round"
      stroke-linejoin="round"
    >
      <path d="M5 12h13M12 5l7 7-7 7" />
    </svg>
  );
};

export const ArrowPointing = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      stroke="%23FFFFFF"
      stroke-width="2"
      stroke-linecap="round"
      stroke-linejoin="round"
    >
      <path d="M5 12h13M12 5l7 7-7 7" />
    </svg>
  );
};
`

File: components/sections/Header.jsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
 import Wrapper from "../wrappers/Wrapper";
-import Button from "../ui/Button";
+import Link from "next/link";
+import Image from "next/image";
+import ArrowSmall from "../../public/assets/images/arrow_small.svg";
```
Comment (esther921) [inline components/sections/Header.jsx]: I suggest you add a more descriptive name to your arrow since we are going to use the arrows from one components

**Review by esther921 — CHANGES_REQUESTED** [review]
The rest code base is fine, except for a few suggestion I have!

**Review by NNesta — APPROVED** [review]
Good Job @iamzilfa

**Review by esther921 — APPROVED** [review]
LGTM

**Review by Stevenkwizera06 — APPROVED** [review]

**Review by Fidesnoella — COMMENTED** [review]
Well done

**Pull request merged**